src/main/java/ch/njol/skript/events/SimpleEvents.java

```diff
@@ -510,5 +510,13 @@
 							"	cancel event")
 					.since("2.3");
 		}
+		if (Skript.classExists("org.bukkit.event.player.PlayerRiptideEvent")) {
+			Skript.registerEvent("Riptide", SimpleEvent.class, PlayerRiptideEvent.class, "[use of] riptide [enchantment]")
```

Comment:
Instead of ``[use of] riptide [enchantment]``, you could do ``[use of] riptide [enchant[ment]]``